application/src/main/java/org/openhds/mobile/utilities/OdkCollectHelper.java

```diff
@@ -149,4 +159,101 @@ public static void setStatusComplete(ContentResolver resolver, Uri uri) {
         resolver.update(uri, cv, null, null);
 
     }
+
+    private static String makePlaceholders(int len) {
+        if (len < 1) {
+            // It will lead to an invalid query anyway ..
+            throw new RuntimeException("No placeholders");
+        } else {
+            StringBuilder sb = new StringBuilder(len * 2 - 1);
+            sb.append("?");
+            for (int i = 1; i < len; i++) {
+                sb.append(",?");
+            }
+            return sb.toString();
+        }
+    }
```

Comment:
Fine to keep private for now I suppose, but I imagine that we may end up wanting to use something like this in other classes. It might make more sense to make it public and factor it out.